Automattic/wp-calypso
# Memberships: add tests for coupons UI

Related to 278-gh-automattic/gold

## Proposed Changes

* Add tests for memberships coupons.

## Testing Instructions



Tests are included. To manual test:
- Stick the appropriate blog sticker to make coupons work on your memberships site. Hint: search wpcom for is_coupons_and_gifts_enabled. This allows the coupons to be used with the backend, otherwise the routes will not exist.
- Sandbox the public API and subscribe.wordpress.com. Enable store sandbox.
- Navigate to /earn/payments/ using one of the calypso live links below.
- Click "Add coupon" and add a coupon. Try various options. Click save.
- Edit the coupons by clicking the ellipsis menu and clicking edit. The same goes for deleting them.
- Create more coupons with different configurations and ensure they work as expected.

## Pre-merge Checklist
